azure-pipelines.yml

```diff
@@ -249,7 +249,7 @@ stages:
             image_tests/bin/pip install -U -r requirements.txt -c constraints.txt
             image_tests/bin/pip install -U -c constraints.txt -e ".[visualization]"
             image_tests/bin/python setup.py build_ext --inplace
-            sudo apt install -y graphviz pandoc
+            sudo apt install -y graphviz pandoc $(cat apt.txt)
```

Comment:
why these is needed? Do this checks run in CI?

Reply:
I think images tests are run in CI at https://github.com/Qiskit/qiskit-terra/blob/0cdacec2ee3b26a96b1d7bbb0def6b7eb66ac5b4/azure-pipelines.yml#L255

Reply:
```suggestion
            sudo apt install -y graphviz pandoc texlive-pictures texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended poppler-utils
```